terra-jupyter-base/custom/edit-mode.js

```diff
@@ -107,7 +107,10 @@ define(() => {
         }
 
         checkMeta();
-        initSyncMaintainer();
+
+        const initialSyncDelay = 27000;
+        //we want subsequent checkMeta + lock calls to be off-cycle of autosave
+        setTimeout(() => initSyncMaintainer(), initialSyncDelay);
```

Comment:
Note: checkMeta will trigger at the following times with this implementation: 0s, 1m27s, 2m27s, ... 

I think this is probably fine (vs also having a check @ 27s), but just calling it out to make sure we're on the same page.

Reply:
Yeah, I feel that its fine to wait a bit after the first call, as such a small difference shouldn't affect the integrity and it makes the code cleaner.